### PR TITLE
Debezium MongoDB Connector native support

### LINKED ISSUE #1190: Debezium MongoDB Connector native support

### PR DESCRIPTION
**Pull request by ppalaga**
fix #1190

Rebased https://github.com/apache/camel-quarkus/pull/2072 and some minor changes to be able to run the tests on Quarkus Platform.

@JiriOndrusek could you please review?

### CODE REVIEW

File: extensions/debezium-mongodb/runtime/pom.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
     <properties>
         <camel.quarkus.jvmSince>1.0.0</camel.quarkus.jvmSince>
-        <camel.quarkus.nativeSince>1.0.0</camel.quarkus.nativeSince>
+        <camel.quarkus.nativeSince>1.5.0</camel.quarkus.nativeSince>
```
Comment (JiriOndrusek) [inline extensions/debezium-mongodb/runtime/pom.xml:34]: From the time of the original PR, version should be bumped to 1.6.

File: pom.xml [hunk 2]
```diff
@@ -56,7 +56,7 @@
         <commons-lang.version>2.6</commons-lang.version><!-- used by hbase, should be pretty stable as commons-lang is not developed actively anymore -->
         <commons-math3.version>3.6.1</commons-math3.version><!-- Mess in the transitive dependencies of Spark and hbase-testing-util -->
         <curator.version>4.3.0</curator.version><!-- Mess in the transitive dependencies of Spark, Zookeeper and other hadoop related components -->
-        <debezium.version>1.3.0.Final</debezium.version>
+        <debezium.version>1.4.0.Beta1</debezium.version>
```
Comment (JiriOndrusek) [inline pom.xml:59]: Here has to be 1.4.0.Final

**Review by JiriOndrusek — CHANGES_REQUESTED** [review]
All new changes look good to me, there are only a few things that has to be changed, because the original PR was old.

File: extensions/debezium-mongodb/runtime/pom.xml (hunk 1 above)
Comment (ppalaga) [reply extensions/debezium-mongodb/runtime/pom.xml:34]: Good catch, thanks!

File: pom.xml (hunk 2 above)
Comment (ppalaga) [reply pom.xml:59]: +1, thanks

**Comment by ppalaga** [comment]
27a8d50: addressed the comments of @JiriOndrusek

**Review by JiriOndrusek — APPROVED** [review]

**Comment by ppalaga** [comment]
The failure in Hazelcast test does not seem to be related.

Thanks, @JiriOndrusek!

**Pull request merged**